lib/util/esWrapper.js

```diff
@@ -124,6 +130,7 @@ class ESWrapper {
    * @returns {KuzzleError}
    */
   formatESError(error) {
+    console.log(error)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)